From fb7b5af1ef98643e84d40e58174d48ba22723253 Mon Sep 17 00:00:00 2001
From: Luca Beltrame <lbeltrame@kde.org>
Date: Fri, 12 Apr 2024 07:51:45 +0200
Subject: [PATCH] Use the openSUSE wallpaper

This is unfortunately hardcoded. Use our default to more likely match what
the user sees.
---
 src/qml/mock/MockDesktop.qml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/qml/mock/MockDesktop.qml b/src/qml/mock/MockDesktop.qml
index d9c906c..19c7b26 100644
--- a/src/qml/mock/MockDesktop.qml
+++ b/src/qml/mock/MockDesktop.qml
@@ -14,12 +14,12 @@ Item {
 
     property int backgroundAlignment: Qt.AlignRight | Qt.AlignBottom
 
-    readonly property string wallpaper: "file:" + Controller.installPrefix() + "/share/wallpapers/Next/contents/images/1280x800.png"
+    readonly property string wallpaper: "file:" + Controller.installPrefix() + "/share/wallpapers/openSUSEdefault/contents/images/1280x1024.png"
     readonly property int desktopWidth: 1280
-    readonly property int desktopHeight: 800
+    readonly property int desktopHeight: 1024
 
     /*
-     * 1280x800 is chosen to look good at normal window sizes: The aspect closely
+     * 1280x1024 is chosen to look good at normal window sizes: The aspect closely
      * matches that of the mock as shown in the application and is not too large.
      *
      * The background be positioned according to the alignment, and expand to fit
-- 
2.44.0

